From c3432ca6336da0ffad1b3daa64b63c026c9ad2e0 Mon Sep 17 00:00:00 2001
From: Kai Willadsen <kai.willadsen@gmail.com>
Date: Sat, 15 Mar 2014 10:57:12 +1000
Subject: [PATCH 1/2] meldapp: Add helper to get the current MeldWindow and use

---
 meld/meldapp.py | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/meld/meldapp.py b/meld/meldapp.py
index cf8efd3..5418085 100644
--- a/meld/meldapp.py
+++ b/meld/meldapp.py
@@ -125,10 +125,13 @@ class MeldApp(Gtk.Application):
         window.widget.meldwindow = window
         return window
 
+    def get_meld_window(self):
+        return self.get_active_window().meldwindow
+
     def open_paths(self, paths, **kwargs):
         new_tab = kwargs.pop('new_tab')
         if new_tab:
-            window = self.get_active_window().meldwindow
+            window = self.get_meld_window()
         else:
             window = self.new_window()
         return window.open_paths(paths, **kwargs)
@@ -251,7 +254,7 @@ class MeldApp(Gtk.Application):
             comparison_file_path = os.path.expanduser(path)
             gio_file = Gio.File.new_for_path(comparison_file_path)
             try:
-                tab = self.window.append_recent(gio_file.get_uri())
+                tab = self.get_meld_window().append_recent(gio_file.get_uri())
             except (IOError, ValueError):
                 parser.error(_("Error reading saved comparison file"))
             return tab
@@ -275,7 +278,7 @@ class MeldApp(Gtk.Application):
                 tab.set_merge_output_file(options.outfile)
 
         if error:
-            if not self.window.has_pages():
+            if not self.get_meld_window().has_pages():
                 parser.error(error)
             else:
                 print(error)
-- 
1.8.3.1

