diff -urN meld-0.9.3/meld.desktop.in meld-0.9.3.patched/meld.desktop.in
--- meld-0.9.3/meld.desktop.in	2004-02-19 17:47:40.000000000 -0500
+++ meld-0.9.3.patched/meld.desktop.in	2004-05-28 10:44:11.258383321 -0400
@@ -5,7 +5,8 @@
 Exec=meld
 Terminal=false
 Type=Application
-Icon=meld
+Icon=meld.png
+StartupNotify=true
 Categories=GNOME;Application;Development;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=meld
