diff -urp meld-1.4.0.OLD/help/C/Makefile meld-1.4.0/help/C/Makefile
--- meld-1.4.0.OLD/help/C/Makefile	2010-09-26 18:25:02.000000000 -0400
+++ meld-1.4.0/help/C/Makefile	2010-10-20 21:24:28.160056001 -0400
@@ -11,21 +11,18 @@ INST_XML := $(helpdir_)/$(LANG)/meld.xml
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
 	-mkdir -m 755 -p $(OMF_DIR_) $(XML_DIR_)
 	install -m 644 meld.xml $(XML_DIR_)/meld.xml
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
 	-rm -f $(OMF_DIR_)/$(OMF_NAME) \
 		$(XML_DIR_)/meld.xml
 	-rmdir $(XML_DIR_)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : clean
 clean :
diff -urp meld-1.4.0.OLD/help/es/Makefile meld-1.4.0/help/es/Makefile
--- meld-1.4.0.OLD/help/es/Makefile	2010-09-26 18:25:02.000000000 -0400
+++ meld-1.4.0/help/es/Makefile	2010-10-20 21:25:09.747056001 -0400
@@ -11,21 +11,18 @@ INST_XML := $(helpdir_)/$(LANG)/meld.xml
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
 	-mkdir -m 755 -p $(OMF_DIR_) $(XML_DIR_)
 	install -m 644 meld.$(LANG).xml $(XML_DIR_)/meld.xml
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
 	-rm -f $(OMF_DIR_)/$(OMF_NAME) \
 		$(XML_DIR_)/meld.xml
 	-rmdir $(XML_DIR_)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : clean
 clean :
diff -urp meld-1.4.0.OLD/help/fr/Makefile meld-1.4.0/help/fr/Makefile
--- meld-1.4.0.OLD/help/fr/Makefile	2010-09-26 18:25:02.000000000 -0400
+++ meld-1.4.0/help/fr/Makefile	2010-10-20 21:25:38.360056002 -0400
@@ -11,21 +11,18 @@ INST_XML := $(helpdir_)/$(LANG)/meld.xml
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
 	-mkdir -m 755 -p $(OMF_DIR_) $(XML_DIR_)
 	install -m 644 meld.$(LANG).xml $(XML_DIR_)/meld.xml
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
 	-rm -f $(OMF_DIR_)/$(OMF_NAME) \
 		$(XML_DIR_)/meld.xml
 	-rmdir $(XML_DIR_)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : clean
 clean :
