diff -urp meld-1.2.1.OLD/help/C/GNUmakefile meld-1.2.1/help/C/GNUmakefile
--- meld-1.2.1.OLD/help/C/GNUmakefile	2007-02-17 13:40:05.000000000 -0500
+++ meld-1.2.1/help/C/GNUmakefile	2008-11-23 10:51:21.000000000 -0500
@@ -11,7 +11,6 @@ INST_XML := $(helpdir_)/$(LANG)/meld.xml
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
@@ -19,7 +18,6 @@ install : $(OMF_NAME).install
 	install -m 644 meld.xml $(XML_DIR_)/meld.xml
 	install -m 644 figures/*.png $(XML_DIR_)/figures
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
@@ -28,7 +26,6 @@ uninstall :
 		$(XML_DIR_)/figures/*.png
 	-rmdir $(XML_DIR_)/figures \
 		$(XML_DIR_)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : clean
 clean :
diff -urp meld-1.2.1.OLD/help/es/GNUmakefile meld-1.2.1/help/es/GNUmakefile
--- meld-1.2.1.OLD/help/es/GNUmakefile	2007-02-17 13:40:05.000000000 -0500
+++ meld-1.2.1/help/es/GNUmakefile	2008-11-23 10:51:54.000000000 -0500
@@ -11,7 +11,6 @@ INST_XML := $(helpdir_)/$(LANG)/meld.xml
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
@@ -19,7 +18,6 @@ install : $(OMF_NAME).install
 	install -m 644 meld.$(LANG).xml $(XML_DIR_)/meld.xml
 	install -m 644 figures/*.png $(XML_DIR_)/figures
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
@@ -28,7 +26,6 @@ uninstall :
 		$(XML_DIR_)/figures/*.png
 	-rmdir $(XML_DIR_)/figures \
 		$(XML_DIR_)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : clean
 clean :
diff -urp meld-1.2.1.OLD/help/fr/GNUmakefile meld-1.2.1/help/fr/GNUmakefile
--- meld-1.2.1.OLD/help/fr/GNUmakefile	2008-03-31 16:44:53.000000000 -0400
+++ meld-1.2.1/help/fr/GNUmakefile	2008-11-23 10:54:18.000000000 -0500
@@ -11,24 +11,17 @@ INST_XML := $(helpdir_)/$(LANG)/meld.xml
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
 	-mkdir -m 755 -p $(OMF_DIR_) $(XML_DIR_) $(XML_DIR_)/figures
 	install -m 644 meld.$(LANG).xml $(XML_DIR_)/meld.xml
-	install -m 644 figures/*.png $(XML_DIR_)/figures
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
 	-rm -f $(OMF_DIR_)/$(OMF_NAME) \
-		$(XML_DIR_)/meld.xml \
-		$(XML_DIR_)/figures/*.png
-	-rmdir $(XML_DIR_)/figures \
-		$(XML_DIR_)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
+		$(XML_DIR_)/meld.xml
 
 .PHONY : clean
 clean :
